altacv.cls

```diff
@@ -38,6 +38,7 @@
 %% v1.3.2 Hopefully this helps make the PDF
 %% file more 'friendly' with copy-paste etc
 \RequirePackage[a-1b]{pdfx}
+\catcode30=12
```

Comment:
Could you please clarify why this is in this PR and not separate? It would be nice to configure only CI/CD in these changes

Reply:
It is needed to compile using the docker image used by the github action. As a fact, we can make another PR.
But this bug will also be fixed in future version xxx (I do not remember that latex package that fails)